Scripts/Python/kdshPillarRoom.py

```diff
@@ -531,13 +525,23 @@ def DisableAppropriateLadders(self,pillar): # only the ladders boxes affected by
         #~ PtDebugPrint(" ladderstodisable = ", ladderstodisable)
 
         for each in ladderstodisable:
-            code = "Ladderbox" +str(each) + ".disable()"
-            exec(code)
+            globals()["Ladderbox{}".format(each)].disable()
             #~ PtDebugPrint("Immediately disabling ladderbox", each)
 
     def EnableAppropriateLadders(self,id): # when pillar is done raising, enable appropriate ladder boxes with new lengths
         ageSDL = PtGetAgeSDL()
-        
+
+        def _ConfigLadder(ladderId, enable=None, deltaH=None):
+            assert enable is not None
```

Comment:
Then why is it defaulted to `None` in the function signature?  ;)

Reply:
whoops. leftover from an old rev